Kitodo-DataManagement/src/main/java/org/kitodo/data/database/beans/Folder.java

```diff
@@ -29,8 +42,13 @@
 import javax.persistence.Table;
 import javax.persistence.Transient;
 
+import org.apache.myfaces.util.FilenameUtils;
+import org.kitodo.api.filemanagement.FileManagementInterface;
 import org.kitodo.api.imagemanagement.ImageManagementInterface;
-import org.kitodo.forms.FolderGenerator;
+import org.kitodo.config.Config;
+import org.kitodo.helper.LexicographicalOrder;
+import org.kitodo.serviceloader.KitodoServiceLoader;
```

Comment:
How can you use the serviceloader from the DataManagement module?
As I see, the kitodo-api was added by you to the dataManagement. this is NOT intendet, as the dataManagement should be independent.

Reply:
This requires the data management being removed out of the form classes, which did not yet happen. This is inheritance of the legacy code.

Reply:
Data Management will not be removed out of form classes as it carries bean objects.... Those functionalities which you implement here, shouldn't be implemented here. Design decision made 2 years ago was that this module is only for data objects, no logic inside.

Reply:
But implementation does not yet follow this decision. So what shell I do? This should have been rewritten with the new front-end, but it wasn’t. If you have a solution for this, let me know.

Reply:
It was following until you broke it for implementation UGH API (https://github.com/kitodo/kitodo-production/pull/1288). We allowed to make API import until UGH is not removed - right after API dependency will be removed from Data Management module. 

This PR can't use this ServiceLoader, as it is not allowed to add any additional logic to Data Management module.

Reply:
Do you have a solution that is ok as long as front-end classes are still missing?

Reply:
Implement it in Core, Task bean takes only get/set.